genomix/genomix-data/src/main/java/edu/uci/ics/genomix/minicluster/GenomixClusterManager.java

```
@@ -283,6 +285,8 @@ private static void startNCs(ClusterType type, int sleepms) throws IOException,
         LOG.info("Starting NC's");
         String startNCCmd = System.getProperty("app.home", ".") + File.separator + "bin" + File.separator
                 + "startAllNCs.sh " + type;
+        Files.setPosixFilePermissions(FileSystems.getDefault().getPath(startNCCmd),
```

I don't see the try/catch?  Or will it fail silently?